File: Fastframe/src/main/java/com/ysl/fastframe/interfaces/IActivityHelper.java
package com.ysl.fastframe.interfaces;

import android.app.Activity;

/**
 * 业务实现此接口，相当于重写BaseActivity
 */
public interface IActivityHelper {
    void onCreate(Activity activity);
    void onStart(Activity activity);
    void onResume(Activity activity);
    void onPause(Activity activity);
    void onStop(Activity activity);
    void onDestroy(Activity activity);
}

File: Fastframe/src/main/java/com/ysl/fastframe/interfaces/IActivityHelper.kt
package com.ysl.fastframe.interfaces

import android.app.Activity

/**
 * 业务实现此接口，相当于重写BaseActivity
 */
interface IActivityHelper {
    fun onCreate(activity: Activity?)
    fun onStart(activity: Activity?)
    fun onResume(activity: Activity?)
    fun onPause(activity: Activity?)
    fun onStop(activity: Activity?)
    fun onDestroy(activity: Activity?)
}